src/main/steps/c100-rebuild/miam/urgency/content.ts

```diff
@@ -79,9 +70,10 @@ export const form: FormContent = {
   fields: {
     miam_urgency: {
       id: 'miam_urgency',
-      type: 'checkboxes',
-      hint: l => l.select_all_apply,
-      validator: value => atLeastOneFieldIsChecked(value),
+      type: 'radios',
+      label: l => l.whyIsApplicationUrgent,
```

Comment:
Is there a connection between 74 and 75? it needs to be govuk-m

Reply:
Yes, labelSize m indicates label to be rendered with medium font size which is same as govuk-m class